include/plugin_interface/SC_Graph.h

```diff
@@ -35,6 +35,7 @@ struct Graph
 	uint32 mNumControls;
 	float *mControls;
 	float **mMapControls;
+	int32 *mAudioBusOffsets;
```

Comment:
this looks like an ABI change. please increment https://github.com/supercollider/supercollider/blob/master/include/plugin_interface/SC_InterfaceTable.h#L24